rpc/src/v1/helpers/errors.rs

```diff
@@ -423,6 +423,9 @@ pub fn transaction_message(error: &TransactionError) -> String {
 		TooCheapToReplace => {
 			"Transaction gas price is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce.".into()
 		}
+		TxTooCheapToReplace { prev, new } => {
```

Comment:
We already have `TooCheapToReplace` error, do you think we can replace the old one with the new one?

Reply:
The old one is constructed here: 

https://github.com/paritytech/parity-ethereum/blob/6c7d0fef4e30e03920a174eea0344260d59a1077/miner/src/pool/queue.rs#L594-L598

from `transaction_pool::Error` which holds the transaction hash, not the gas prices

https://github.com/paritytech/parity-common/blob/57555f49b6873193437624404121f17adae2723b/transaction-pool/src/error.rs#L21-L28

Reply:
Maybe we could make the `prev` and `new` `Option`al, so that we print more info if it's available, while keeping one error type. Or just document while there are two error types for the same error.